src/amber/controller/render.cr

```diff
@@ -37,5 +37,43 @@ module Amber::Controller
         Kilt.render("#{{{path}}}/{{filename.id}}")
       {% end %}
     end
+
+    protected def respond_with(html : String? = nil, json : Hash | String? = nil, xml : String? = nil, text : String? = nil)
+      accepts = context.request.headers["Accept"].split(";").try(&.split(/,|,\s/))
+
+      if accepts.includes?("text/html") && html
```

Comment:
Can this be a constant if you want to keep the string as such:

```crystal
RESPON_WITH = { 
  html: "text/html", 
  json: "application/json", 
  text: "text/plain", 
  xml: "application/xml"
}
```

Reply:
Yeah that looks good.